jdk-annotations/test-annotated/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id "java-library"
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(21)
+    }
+}
+
+dependencies {
+    compileOnly deps.apt.autoService
+    annotationProcessor deps.apt.autoService
+    compileOnly project(":nullaway")
+    api deps.build.jspecify
+    implementation project(":jdk-annotations:astubx-generator-cli")
+
+    // Add the jdk-javac-plugin as annotation processor
+    annotationProcessor files("${rootProject.projectDir}/jdk-javac-plugin/build/libs/jdk-javac-plugin-all.jar")
```

Comment:
You can't just write `annotationProcessor project(":jdk-javac-plugin:shadowJar")`?